### ISSUE TITLE
Consumption, View log and Parameters empty?

### ISSUE THREAD

**Issue by Nomisdk86** [motoz/PellMon#123]
Hello

I have been running PellMon for 2-3 months and noticed that i no longer can see any data under Consumption, Parameters and the View log.
Today i did try to reset the rrd.db but dont seem to have fixed it.

I have restarted the pellmonsrv/web multi times but no changes.

Consumption:
![image](https://github.com/motoz/PellMon/assets/74494621/c00c5b97-6bf6-4c1c-8455-4331af2e0fb0)

Unable to see settings:
![image](https://github.com/motoz/PellMon/assets/74494621/82915e4d-a07a-45a6-9ea7-f95e6e74d204)

View Log:
![image](https://github.com/motoz/PellMon/assets/74494621/726a2a77-2030-4f5a-b6c6-d75e63494824)

The strange thing is that Pellmon seems to talk correctly to the Scotte, at it the front page displays correctly:
![image](https://github.com/motoz/PellMon/assets/74494621/5dfecfda-9b0c-423c-be37-2eab1fa8c860)

How do i fix this?

**Comment by motoz** [comment]
Just a long shot, try disabling any adblocker or similar. It has happened that it has blocked the event view, so it is not impossible.

**Comment by Nomisdk86** [comment]
Thanks, it was PiHole blocking that did it ajax.googleapis.com.
Is there any way to fill out the missing consumption info (after i have recreated the db)
I remember how much there was used each last month.

**Comment by motoz** [comment]
Not really. The rrd database doesn't hold monthly values at all, it calculates them on the fly from the data that is saved every 30 seconds and then consolidated into coarser data as it gets older. Theoretically it is of course possible to feed these data points with fake timestamps with a custom made script using the average 30s consumtion values for the missing months, but you'd need to know a lot of how the rrd database works.
